internal/controllers/topology/cluster/conditions_test.go

```diff
@@ -49,32 +50,165 @@ func TestReconcileTopologyReconciledCondition(t *testing.T) {
 		name                        string
 		reconcileErr                error
 		s                           *scope.Scope
-		cluster                     *clusterv1.Cluster
 		machines                    []*clusterv1.Machine
-		wantConditionStatus         corev1.ConditionStatus
+		wantV1Beta1ConditionStatus  corev1.ConditionStatus
+		wantV1Beta1ConditionReason  string
+		wantV1Beta1ConditionMessage string
+		wantConditionStatus         metav1.ConditionStatus
 		wantConditionReason         string
 		wantConditionMessage        string
-		wantV1Beta2ConditionStatus  metav1.ConditionStatus
-		wantV1Beta2ConditionReason  string
-		wantV1Beta2ConditionMessage string
 		wantErr                     bool
 	}{
+		// Reconcile error
```

Comment:
This case here is paused and should belong to the next section

l.129++ should be here